### DISCUSSION TITLE
Should we consider renaming the "examples" repo as "miden playground"?

### DISCUSSION THREAD

**Discussion by gobidash** [0xPolygonMiden/examples#196] [General]
I see Miden Playground becoming a module by itself like Miden Client or Miden Node. And `examples` can be brought under Miden Playground in the repository structure - just like how it is actually in the Playground.

Additionally, renaming `examples` into `Miden Playground` will also fit well with the naming convention of other main repos - it is slightly odd right now.
Following this, we can use this repo to host the `miden.io` site. as we will have to make it more dynamic in future, for example adding `miden.io/apps` and probably a forum to the site. As of now we are only redirecting `miden.io` to `https://0xpolygonmiden.github.io/examples/`
cc: @bobbinth @Dominik1999

**Comment by Dominik1999** [comment]
We now have playground.miden.io and explorer.miden.io

**Reply by gobidash** [reply]
Yes, but my suggestion is about the repo not the domain :)

**Reply by gobidash** [reply]
We can also consider adding ecosystem.miden.io or miden.io/ecosystem probably once we have a good number of apps/games. 
On this page, we can include Oracle(s), Wallet(s), Apps, Games, etc.,

**Reply by bobbinth** [reply]
> Yes, but my suggestion is about the repo not the domain :)

I would probably keep this repo as is but create another repo for the playground which would be more rollup-oriented.
